.ci/go-lint.sh

```diff
@@ -25,16 +25,15 @@ then
 fi
 
 gometalinter \
-	--exclude='error return value not checked.*(Close|Log|Print).*\(errcheck\)$' \
-	--exclude='.*_test\.go:.*error return value not checked.*\(errcheck\)$' \
-	--exclude='duplicate of.*_test.go.*\(dupl\)$' \
-	--disable=aligncheck \
-	--disable=gotype \
-	--disable=gas \
-	--disable=vetshadow \
-	--cyclo-over=15 \
 	--tests \
-	--deadline=600s \
 	--vendor \
-	--errors \
+	--disable-all \
```

Comment:
Yes, but if we disable all linters and then only enable what we want (and crucially know about), how will we know when new linters (that we may wish to use) become available? My suggested change will allow developers locally to become aware of new linters. Yes, the lint may fail but that's what we want as it will trigger the question in the developers mind, "do we need to enable this new linter for the CI?". And if they frankly don't care, they can simply do:

```
(CI=true .ci/go-lint.sh) 
```